docs/articles/clustering/distributed-publish-subscribe.md

```diff
@@ -30,7 +30,7 @@ You register actors to the local mediator with `DistributedPubSubMediator.Subscr
 
 You publish messages by sending `DistributedPubSubMediator.Publish` message to the local mediator.
 
-Actors are automatically removed from the registry when they are terminated, or you can explicitly remove entries with `DistributedPubSubMediator.Unsubscribe`.
+Topic Actors are automatically removed from the registry when they are terminated - they are terminated if there are zero existing subscribers and no new subscription within a given deadline (the default is 2 minutes). You can change the deadline by setting `removed-time-to-live` to a desired duration. You can `Unsubscribe` from a topic with `DistributedPubSubMediator.Unsubscribe`.
 
 An example of a subscriber actor:
 
```

Comment:
Need to turn this into a DocFx ref

Reply:
All code samples in this doc need to be updated

Reply:
Good suggestion cc @eaba

Reply:
Note taken!